ProcessLib/NumericsConfig.h

```diff
@@ -65,12 +77,20 @@ namespace detail
 #endif    // USE_LIS
 #endif    // OGS_USE_EIGEN
 
-
 //
 // Global vector/matrix builder.
 //
 
-#include "AssemblerLib/SerialVectorMatrixBuilder.h"
+#ifdef USE_PETSC
```

Comment:
The ifdef and the PETScVectorMatrixBuilder are not necessary. The SerialVectorMatrixBuilder will forward all arguments to the constructors of matrix and vector.
Do I overlook something obvious?